### ISSUE TITLE
Clarification on dependencies verification

### ISSUE THREAD

**Issue by k88ydid** [tandonneur/AdvancedAnalytics#4]
> conda list nltk
> conda list sci-learn
> conda list wordcloud

When I type 'conda list sci-learn' into my prompt, I don't get any results. When I searched for solutions online, I'm most often directed to solutions for installing 'scikit-learn' instead. Just curious if this is a typo in the dependencies section? If not, what is the correct command to install 'sci-learn' if it did not automatically install with AdvancedAnalytics?

**Comment by tandonneur** [comment]
Yes typo should beConda list scikit-learnI think 🤔 Sent from my iPhoneOn Sep 1, 2024, at 11:15, Caitlin Tyson ***@***.***> wrote:﻿

conda list nltk
conda list sci-learn
conda list wordcloud

When I type 'conda list sci-learn' into my prompt, I don't get any results. When I searched for solutions online, I'm most often directed to solutions for installing 'scikit-learn' instead. Just curious if this is a typo in the dependencies section? If not, what is the correct command to install 'sci-learn' if it did not automatically install with AdvancedAnalytics?
